Backport that fixes `make install' with Qt 5.9:

    /usr/local/lib/qt5/bin/qmake -install qinstall /wrkdirs/usr/ports/games/gottet/work/gottet-1.1.3/translations/*.qm /wrkdirs/usr/ports/games/gottet/work/stage/usr/local/share/gottet/translations/*.qm
    Error: usage: [-exe] source target
    *** Error code 3 (ignored)

From e299632494d3bfbbb4833c3feb7a3d7def77dd34 Mon Sep 17 00:00:00 2001
From: Graeme Gott <graeme@gottcode.org>
Date: Sat, 1 Jul 2017 04:54:46 -0400
Subject: [PATCH] FIXED: Did not always install translations in Linux.

---
 gottet.pro | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/gottet.pro b/gottet.pro
index 8a337c4..aa8591d 100644
--- gottet.pro
+++ gottet.pro
@@ -80,7 +80,7 @@ macx {
 	appdata.files = icons/gottet.appdata.xml
 	appdata.path = $$PREFIX/share/metainfo/
 
-	qm.files = translations/*.qm
+	qm.files = $$replace(TRANSLATIONS, .ts, .qm)
 	qm.path = $$PREFIX/share/gottet/translations
 	qm.CONFIG += no_check_exist
 
